### ISSUE TITLE
Ability to create ViewComposers

### ISSUE THREAD

**Issue by andrey-helldar** [laravel-idea/plugin#9]
It would be cool to add the ability to create `ViewComposers` files.

Place: `app/Http/View/Composers` ([see](https://laravel.com/docs/6.x/views#view-composers)).

Template:
```php
<?php

namespace App\Http\View\Composers;

use Illuminate\View\View;

class MyComposer
{
    public function compose(View $view)
    {
        // $view->with('foo', 'bar');
    }
}
```

**Comment by adelf** [comment]
Sure. I'm only not sure about namespace. Why do we need this empty `View` namespace? I usually use `App\Http\ViewComposers` namespace. Of course everyone can change it, but still...

**Comment by andrey-helldar** [comment]
I also always use `App\Http\ViewComposers`, but this is the exact path indicated in the documentation. Therefore, he proposed.

**Comment by andrey-helldar** [comment]
The general [vote](https://t.me/laravel_web/175377) decided to use the namespace `App\Http\ViewComposers`.

**Comment by adelf** [comment]
Implemented in 2.0. Thanks.

**Comment by andrey-helldar** [comment]
Thanks! 🍻
